tcti/tcti.h

```diff
@@ -100,6 +100,7 @@ typedef enum {
     TCTI_STATE_FINAL,
     TCTI_STATE_TRANSMIT,
     TCTI_STATE_RECEIVE,
+    TCTI_STATE_CANCEL,
```

Comment:
You've added a new state to the state machine but you've added no information about what the state is for, how we get into this state or how we get out of it.

Reply:
Actually a new state wasn't the best idea. I have added a new flag which indicates if a previous command has been cancelled. I have added a test to cover this scenario and when paying with it I noticed that we faked a CANCEL rc from TPM into something else. This has been fixed also now.